docs/data/material/components/autocomplete/Asynchronous.js

```diff
@@ -14,44 +14,30 @@ function sleep(duration) {
 export default function Asynchronous() {
   const [open, setOpen] = React.useState(false);
   const [options, setOptions] = React.useState([]);
-  const loading = open && options.length === 0;
```

Comment:
We received feedback in slack, that if empty array is returned [here](https://github.com/mui/material-ui/pull/43630/files#diff-6ec4be473dc14372f85fa1453db0a718fcadbbe798c66a0bb1611251c0989717R26) demo is in `loading` state indefinetaly. So i decoupled `loading` state `options`